backend/src/dao/deletePending.ts

```diff
@@ -0,0 +1,25 @@
+import * as firebase from 'firebase-admin';
+import { teamEventAttendanceCollection } from '../firebase';
+// import { createStatusFromPending } from '../API/teamEventsAPI';
```

Comment:
delete comment

Reply:
I was executing this file to modify the dev database. I'll remove it entirely once the changes are up on prod.